src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj

```diff
@@ -23,4 +26,7 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.6.0" />
```

Comment:
@jaczhan you do not need this, this is being added to every project.
Any reason you had to add reference to TestFramework?